### DISCUSSION TITLE
About keyboard settings in Windows Access

### DISCUSSION THREAD

**Discussion by mukitaro** [gravitational/teleport#45578] [Q&A] [answered]
It seems that the keyboard setting is set to US when using Windows Access.
Is it possible to change to the keyboard setting I am using?

**Comment by zmb3** [comment, marked as answer]
Not yet, but it's on the roadmap. Feel free to subscribe to #41291 for updates.

**Comment by mukitaro** [comment]
Thanks for the answer, I understand.
